vpp-manager/images/init-eks/init_eks.sh

```diff
@@ -88,11 +113,14 @@ configure_machine ()
 		build_and_install_igb_uio
 		sudo insmod /lib/modules/$(uname -r)/kernel/drivers/uio/igb_uio.ko wc_activate=1
 	fi
+
+	# configure hugepages and persist the config across reboots
 	sudo sysctl -w vm.nr_hugepages=${HUGEPAGES}
 	if [ -f /sys/fs/cgroup/hugetlb/kubepods/hugetlb.2MB.limit_in_bytes ]; then
 		echo $((HUGEPAGES * 2 * 1024 * 1024)) | tee /sys/fs/cgroup/hugetlb/kubepods/hugetlb.2MB.limit_in_bytes
 	fi
-	systemctl restart kubelet
```

Comment:
This might be needed if we want to keep the script (to update hugepage count in kubelet) - if it has not been fixed -

Reply:
Good point. We do want to keep the script. It is needed for the cloud-init/userdata wrapper script and it could also be useful as a standalone script for debugging purposes. And yes, kubelet needs to be restarted when/if we run the script standalone.